framework/packages/abstract-core/src/native/ibc_client.rs

```diff
@@ -106,33 +105,18 @@ pub enum ExecuteMsg {
         host_chain: String,
         // execute the custom host function
         action: HostAction,
-        // optional callback info
-        callback_info: Option<CallbackInfo>,
-    },
-    /// Allows to query something on a remote contract and act on that query result
-    /// This has to be an Execute variant for IBC queries
-    RemoteQueries {
```

Comment:
Why are we removing queries all together?

Reply:
Because queries need callbacks and we agreed that callbacks can't be authentified so we can't use queries